weblate/utils/checks.py

```diff
@@ -354,15 +354,15 @@ def check_site(app_configs, **kwargs):
 
     errors = []
     if not check_domain(get_site_domain()):
-        errors.append(weblate_check("weblate.E017", "Configure correct site domain"))
+        errors.append(weblate_check("weblate.E017", "Correct the site domain"))
     return errors
 
 
 def check_perms(app_configs=None, **kwargs):
-    """Check we can write to data dir."""
+    """Check that the data dir can be written to."""
     errors = []
     uid = os.getuid()
-    message = "Path {} is owned by different user, check your DATA_DIR settings."
+    message = "The path {} is owned by different user, check your DATA_DIR settings."
```

Comment:
Should {} be quoted?

Reply:
It's file system path, probably not necesary to quote it...